Create ui-c8y-1021-54-7-in-memory-temporary-dashboard-for-users-without-permissions.md (#3104)
SoftwareAG/c8y-docs: content/change-logs/device-management/ui-c8y-1021-54-7-in-memory-temporary-dashboard-for-users-without-permissions.md

---
date: ""

> title: Allow users without permissions to see a fallback temporary dashboard

Reviewer: ```suggestion
title: Display read-only dashboard for users without dashboard creation permission
```

> title: Allow users without permissions to create temporary dashboards in memory

Reviewer 2: ```suggestion
title: Allow users without permissions to see a fallback temporary dashboard
```

product_area: Device management & connectivity
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component--KIsStyzM
    label: Device Management app
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: DM-4362
version: 1021.54.7
---

> When users lack dashboard creation permissions, a temporary read-only dashboard is displayed to enable data visualization and prevent a blank page. This fallback dashboard is an in-memory one and not persisted in the backend.

Reviewer: ```suggestion
When users do not have the permission to create dashboards, a temporary read-only dashboard is now displayed to enable data visualization and prevent a blank page. This fallback dashboard is an in-memory one and not persisted in the backend.
```

> To enable users to explore and visualize data without requiring explicit dashboard creation permissions, a new feature has been added that allows users to create temporary dashboards that are stored in memory. Previously, users needed specific permissions to create and save dashboards. With this change, any user can now create temporary dashboards to visualize and analyze data, but these dashboards are not persisted and will be lost when the user session ends. This provides more flexibility for users to work with data without impacting existing permission models or dashboard assets.

Reviewer 2: ```suggestion
When users lack dashboard creation permissions, a temporary read-only dashboard is displayed to enable data visualization and prevent a blank page. This fallback dashboard is an in-memory one and not persisted in the backend.
```
